<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Bot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1. Open URL / id - wait 5 secs&lt;/p&gt;&lt;p&gt;2. check pixel&lt;/p&gt;&lt;p&gt;- for sellinng , CTRL+W&lt;/p&gt;&lt;p&gt;- else CTRL+W&lt;/p&gt;&lt;p&gt;3. Open URL / id / sell - wait 5 secs&lt;/p&gt;&lt;p&gt;4. INPUT 0.0041&lt;/p&gt;&lt;p&gt;5. CLICK Duration, Date Range, 6 months, CLICK outside, &lt;/p&gt;&lt;p&gt;6. Click complete Listing - wait 5 secs&lt;/p&gt;&lt;p&gt;7. Click Scroll down on Metamask&lt;/p&gt;&lt;p&gt;8. Click Sign on Metamask - wait 2 secs&lt;/p&gt;&lt;p&gt;9. Close Browser&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pickPosButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick Position for Checking selling</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startNftId">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="initalPts">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="plainText">
     <string>820,379
1033,162
187,331
274,399
216,529
61,365
207, 478
1553,519
1465,580</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spdSync">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pickPosButton</zorder>
   <zorder>startNftId</zorder>
   <zorder>initalPts</zorder>
   <zorder>spdSync</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
